charts/kueue/values.yaml

```diff
@@ -9,6 +9,15 @@ enablePrometheus: false
 enableCertManager: false
 # Customize controlerManager
 controllerManager:
+  featureGates:
```

Comment:
IIUC this overrides the defaults specified in code: https://github.com/kubernetes-sigs/kueue/blob/b47bf0327aae21e6f512dcd33c2b89bdc2044347/pkg/features/kube_features.go#L71-L74

I'm wondering if this will start to require us to remember to modify both places on graduations.

If this section is just to demonstrate how to override maybe we can have it just for one feature gate and commented out, something like here: https://github.com/kubernetes-sigs/kueue/blob/91d5f645a326dae1ea8542879adcc62ba4db7f5d/charts/kueue/values.yaml#L81-L84

Reply:
+1, leave one example here and comment them out as a reference.

Reply:
agree, done.